src/eth/rpc/tx.h

```diff
@@ -15,9 +15,13 @@ struct tx {
   bytes20 to;
   uint256_t value;
   bytes data;
-  uint8_t v;
+  uint256_t v;
```

Comment:
```suggestion
  uint64_t v;
```
Isn't 64bit enough this time?